kafka/README.md

```diff
@@ -144,17 +144,65 @@ az role assignment create --role "Key Vault Crypto User" --assignee <alias>@micr
 
 NOTE: Only the subscription owner can setup role access for AKV/mHSM, so if you are seeing authorization related error messages during role access setup steps, please seek out the proper personel to setup role access. 
 
-#### Install Kafka Cluster 
+
+#### Create Kafka Cluster 
+
+**NOTE**: Due to security issues, we have deprecated running Kafka cluster on k8s using Strimzi. The preferred kafka cluster in this example is Microsoft provisioned Azure Event Hub. However, we still keep the old instruction on how to run this demo using Kafka cluster on k8s using Strimzi. See the following: 
```

Comment:
nit: maybe we could link them to a doc introducing Event Hub or add a little more introduction to it since it's another Microsoft product. 

Like maybe replace
> The preferred kafka cluster in this example is Microsoft provisioned Azure Event Hub.
with 
> Instead, we recommend using [Azure Event Hub](https://learn.microsoft.com/en-us/azure/event-hubs/event-hubs-about), a fully managed, real-time data ingestion service that's highly compatible with Kafka. The instructions below guide you through setting up a Kafka-compatible Azure Event Hub.